// try {
//         throw "Error en tipo de datos"
// } catch (e) {
//     console.log('Error en llamado: ', e);
// }

try {
    throw "Error en tipo de datos"
} catch {
    console.log('Error en llamado a la rutina');
}